app/modules/softuart.c

```diff
@@ -23,19 +24,15 @@ typedef struct {
 } softuart_buffer_t;
 
 typedef struct {
-	uint8_t pin_rx;
-	uint8_t pin_tx;
 	volatile softuart_buffer_t buffer;
 	uint16_t bit_time;
 	uint16_t need_len; // Buffer length needed to run callback function
 	char end_char; // Used to run callback if last char in buffer will be the same
 	uint8_t armed;
+	uint8_t pin_rx;
+	uint8_t pin_tx;
```

Comment:
Why were these moved?  Does it reduce padding within the structure?

Reply:
This was the intent - I'm not sure if this would help, but having the buffer as a first element of the struct should make access to it a bit easier and maybe instruction or two faster - but I haven't tested it.